addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -271,23 +271,24 @@ DebugManager.include({
         });
     },
     get_view_fields: function () {
-        var self = this;
-        var model = this._action.res_model;
-        this._rpc({
-                model: model,
-                method: 'fields_get',
-                kwargs: {
-                    attributes: ['string', 'searchable', 'required', 'readonly', 'type', 'store', 'sortable', 'relation', 'help']
-                },
-            })
-            .done(function (fields) {
-                new Dialog(self, {
-                    title: _.str.sprintf(_t("Fields of %s"), model),
-                    $content: $(QWeb.render('WebClient.DebugManager.Action.Fields', {
-                        fields: fields
-                    }))
-                }).open();
+        var model = this._action.res_model,
+            self = this;
+        self._rpc({
```

Comment:
I can't find it in the guidelines and it's nitpicking, but please use `self` only when you change scope (so, use `this` whenever possible)